pnc-mock/src/main/java/org/jboss/pnc/mock/repository/BuildRecordRepositoryMock.java

```diff
@@ -166,9 +154,7 @@ public static BuildRecord getLatestSuccessfulBuildRecord(
                 .filter(
                         buildRecord -> buildRecord.getBuildConfigurationAuditedIdRev()
                                 .equals(buildConfigurationAuditedIdRev))
-                .filter(
-                        buildRecord -> (temporaryBuild && buildRecord.isTemporaryBuild())
-                                || !buildRecord.isTemporaryBuild())
+                .filter(buildRecord -> temporaryBuild || !buildRecord.isTemporaryBuild())
```

Comment:
I just noticed the condition at line 95 is the same, just with applied De Morgan's rule. It would be nice to have the same one everywhere.